ls1intum/Artemis
# Specify chai expect and icon/text usage guidelines

### Checklist

### Motivation and Context
1. Chai gives some suggestions in their documentation (as linked in the changes) or to be precise recommends not doing certain things that we do.
2. We have different ways to enforce a margin between icons and text in buttons and want to unify it.

### Description
1. I described the two most important takeaways and fixed one occurrence in the documentation that uses the bad practice
2. As described, HTML already converts one or multiple newlines to one space which gets already used in a lot of occurrences. If we would use styling as originally suggested, we would get issues if the tags get long due to long attribute values. Therefore, I decided to enforce the newline instead.

This only changes the guidelines. Part of 1) could be changed at once in another PR. 2) Could also be adapted in another PR by searching all icon occurences.

### Steps for Testing
Just check the documentation. I also hope that the documentation is correct since I can't view it locally.

### Review Progress





- Code Review
